libs/Microsoft.MixedReality.WebRTC/RemoteVideoTrack.cs

```diff
@@ -124,6 +124,8 @@ public bool Enabled
         private event I420AVideoFrameDelegate _videoFrameReady;
         private event Argb32VideoFrameDelegate _argb32VideoFrameReady;
 
+        public IntPtr NativeHandle => _nativeHandle.DangerousGetHandle();
```

Comment:
It would be better to expose a `SafeHandle` instead - that will keep the handle to the native object alive. Passing around the `IntPtr` will not.

Also this needs a comment specifying it's internal use only.

Reply:
So I should return the `RemoteVideoTrackInterop.RemoteVideoTrackHandle` ? That would require `RemoteVideoTrackHandle` and `RemoteVideoTrackInterop` to become public. Which could happen, but the purpose of this property is so that the Unity plugin can retrieve the native pointer and send it down to C++.  The Unity plugin which retrieves this does not itself hold onto the managed pointer, it immediately passes it to C++, so it would not benefit from the SafeHandle keeping the object alive. Which with that, do you think it should still be changed to return `RemoteVideoTrackHandle`?

Reply:
That's a good point. Since this is a public property I'd say it's still better to be safe than sorry and return a `SafeHandle`, just to prevent bugs if some code at some point decides to keep this around, but you can also just add a "Internal use" comment and it should be fine for now. Definitely not worth exposing `RemoteVideoTrackHandle`.